app/views/content/life-as-a-teacher/why-teach/why-become-a-teacher/_article.html.erb

```diff
@@ -18,6 +18,8 @@
 
     <p><%= render 'content/shared/quotes/quote_why_teach_helen' %></p>
 
+    <p>Find out more about <a href="/steps-to-become-a-teacher">how to become a teacher</a>.</p>
```

Comment:
This feels slightly odd under the creativity section?

Would it be better under a new section at the end of the page with its own heading?